pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -44,11 +44,12 @@ import (
 const (
 	// reconcilerLoopSleepPeriod is the amount of time the reconciler loop waits
 	// between successive executions
-	reconcilerLoopSleepPeriod time.Duration = 100 * time.Millisecond
+	reconcilerLoopSleepPeriod        time.Duration = 100 * time.Millisecond
+	reconcilerReconstructSleepPeriod time.Duration = 3 * time.Minute
 
 	// desiredStateOfWorldPopulatorLoopSleepPeriod is the amount of time the
 	// DesiredStateOfWorldPopulator loop waits between successive executions
-	desiredStateOfWorldPopulatorLoopSleepPeriod time.Duration = 100 * time.Millisecond
+	desiredStateOfWorldPopulatorLoopSleepPeriod time.Duration = 500 * time.Millisecond
```

Comment:
Let's leave this at the current rate, unless we have a good reason to change it.